Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -267,8 +270,7 @@ - (void)configureNavigationButtons {
 - (void)configureTableView {
     NSAssert(_tableView == nil, @"_tableView is already initialized!");
 
-    self.tableView = [[SPBorderedTableView alloc] init];
-    self.tableView.frame = self.rootView.bounds;
+    self.tableView = [[UITableView alloc] initWithFrame:self.view.bounds style:UITableViewStylePlain];
```

Comment:
Definitely a good idea. Since we're using the `Plain` style, and we're actually using autolayout (nowadays!!!), i'm switching over to the plain old `[UITableView new]` .

Thank you!